alibi_detect/saving/tests/models.py

```diff
@@ -64,12 +70,15 @@ def encoder_dropout_model(backend, current_cases):
                [
                    tf.keras.layers.InputLayer(input_shape=(input_dim,)),
                    tf.keras.layers.Dense(5, activation=tf.nn.relu),
-                   tf.keras.layers.Dropout(0.5),
+                   tf.keras.layers.Dropout(0.0),  # 0.0 to ensure determinism
```

Comment:
Unlikely to give a sensible model for `RegressorUncertaintyDrift`, but if > 0.0 detector predictions are not deterministic. OK for testing serialization I think...